src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -95,7 +95,7 @@ export const SplitButton = (props: SplitButtonProps) => {
         onShow={React.useCallback(() => setIsMenuOpen(true), [])}
         onHide={React.useCallback(() => setIsMenuOpen(false), [])}
       >
-        <IconButton styleType={styleType} size={size}>
+        <IconButton styleType={styleType} size={size} disabled={props.disabled}>
```

Comment:
You should also add `iui-disabled` to the span className.

It belongs in this file, rather than the header button